packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@anticrm/core",
-  "version": "0.6.12",
```

Comment:
I'm just curious, should we as well update all local deps? 
Most of them in format '"@anticrm/core": "~0.6.x' or should we have them in format "^0.6.0"?